src/pages/de/guides/data-fetching.md

```diff
@@ -0,0 +1,172 @@
+:::
+
+
+## `fetch()` in Framework-Komponenten
+
+Die `fetch()`-Funktion ist auch global in jeder [Framework-Komponente](/de/core-concepts/framework-components/) verfügbar:
+
+```tsx title="src/components/Movies.tsx" /await fetch\\(.*?\\)/
+import type { FunctionalComponent } from 'preact';
+import { h } from 'preact';
+
+const data = await fetch('https://example.com/movies.json').then((response) =>
+  response.json()
+);
+
+// Komponenten die zum Zeitpunkt der Erstellung gerendert werden, loggen Daten auch in der CLI.
+// Wenn sie mit einer client:*-Direktive gerendert werden, wird dies auch in der Browser-Konsole angezeigt.
+console.log(data);
+
+const Movies: FunctionalComponent = () => {
+// Ausgabe des Ergebnisses in die Seite
+  return <div>{JSON.stringify(data)}</div>;
+};
+
+export default Movies;
+```
+
+
+## GraphQL-Abfragen
+
+Astro kann auch `fetch()` nutzen, um GraphQL-Server mit einer beliebigen gültigen GraphQL-Anfrage anzufragen.
+
+```astro title="src/components/Weather.astro" "await fetch"
+---
+const response = await fetch("https://graphql-weather-api.herokuapp.com",
+  {
+    method: 'POST',
+    headers: {'Content-Type':'application/json'},
+    body: JSON.stringify({
+      query: `
+        query getWeather($name:String!) {
+            getCityByName(name: $name){
+              name
+              country
+              weather {
+                summary {
+                    description
+                }
+              }
+            }
+          }
+        `,
+      variables: {
+          name: "Toronto",
+      },
+    }),
+  });
+
+const json = await response.json();
+const weather = json.data;
+---
+<h1>Abruf des Wetters zur Erstellungszeit</h1>
+<h2>{weather.getCityByName.name}, {weather.getCityByName.country}</h2>
+<p>Wetter: {weather.getCityByName.weather.summary.description}</p>
+```
+
+## Abfragen von einem Headless-CMS
+
+Lade Remote-Inhalte aus deinem favourisiertem CMS, wie Storyblok oder WordPress!
+
+Astro-Komponenten könnten Daten von deinem CMS abrufen und in deinen Seiteninhalt rendern. Mit [dynamischen Routen](//de/core-concepts/routing/#dynamische-routen) können Komponenten sogar Seiten auf der Basis deiner CMS-Inhalte generieren.
+
+Hier sind ein paar Beispiele, wie die Datenabfrage in Astro aussieht, mit Links zu den vollständigen Tutorials unten.
+
+### Beispiel: Storyblok API
+
+```astro
+---
+// src/pages/index.astro
+// Abrufen einer Liste deine Storyblok-Seitenlinks mit @storyblok/js
+import BaseLayout from '../layouts/BaseLayout.astro';
+import { storyblokInit, apiPlugin } from "@storyblok/js";
+
+const { storyblokApi } = storyblokInit({
+  accessToken: "MY_STORYBLOK_ACCESS_TOKEN",
```

Comment:
```suggestion
  accessToken: "MEIN_STORYBLOK_ACCESS_TOKEN",
```

I'd translate this to indicate that it's not some magic keyword, but a simple user-defined string.